Clients/src/presentation/components/Modals/Controlpane/index.tsx

```
@@ -130,59 +130,58 @@ const CustomModal: React.FC<CustomModalProps> = ({
+          {["Overview", "Evidence", "Auditor Feedback"].map((section) => (
+            <Button
+              variant={getVariant(activeSection, section)}
+              onClick={() => handleSectionChange(section)}
+              disableRipple
+              sx={{
+                ...buttonTabStyles,
+                backgroundColor:
+                  activeSection === section ? "#EAECF0" : "transparent",
+                fontWeight: activeSection === section ? "500" : 300,
+              }}
+            >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)